docs/safety.md

```diff
@@ -415,21 +416,14 @@ Regular monitoring of a node and keeping track of the movement of funds can help
 prevent problems. Tools like [`lndmon`](https://github.com/lightninglabs/lndmon)
 can assist with these tasks.
 
-### Pruned bitcoind node
-
-Running `lnd` connected to a `bitcoind` node that is running in prune mode is
-not supported! `lnd` needs to verify the funding transaction of every channel
-in the network and be able to retrieve that information from `bitcoind` which
-it cannot deliver when that information is pruned away.
-
-In theory pruning away all blocks _before_ the SegWit activation would work
-as LN channels rely on SegWit. But this has neither been tested nor would it
-be recommended/supported.
+### The `-txindex` flag
 
 In addition to not running a pruned node, it is recommended to run `bitcoind`
```

Comment:
Perhaps we should re-word this now? That for performance reasons it is recommended to _not_ run a pruned node if possible and to enable the `-txindex` flag as well?